modules/autoattach.cpp

```diff
@@ -116,22 +118,36 @@ class CChanAttach : public CModule {
         }
     }
 
+    void HandleSwap(const CString& sLine) {
+        u_int iA = sLine.Token(1).ToUInt();
+        u_int iB = sLine.Token(2).ToUInt();
+
+        Swap(iA, iB);
+    }
+
     void HandleList(const CString& sLine) {
+        int Index;
         CTable Table;
+        Table.AddColumn(t_s("Index"));
         Table.AddColumn(t_s("Neg"));
         Table.AddColumn(t_s("Chan"));
         Table.AddColumn(t_s("Search"));
         Table.AddColumn(t_s("Host"));
 
+        Index = 1;
         VAttachIter it = m_vMatches.begin();
         for (; it != m_vMatches.end(); ++it) {
             Table.AddRow();
+            Table.SetCell(t_s("Index"), to_string(Index));
             Table.SetCell(t_s("Neg"), it->IsNegated() ? "!" : "");
             Table.SetCell(t_s("Chan"), it->GetChans());
             Table.SetCell(t_s("Search"), it->GetSearch());
             Table.SetCell(t_s("Host"), it->GetHostMask());
+            Index += 1;
         }
 
+        delete Index;
```

Comment:
WAT?

Reply:
Sorry, my C++ is really rusty. Will push with fix soon.